src/components/wordLoader/wordLoader.test.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { Icon, Button, TextArea } from "semantic-ui-react";
+
+import { WordLoader } from "./index";
```

Comment:
We can simply import  from the index as: `import { WordLoader } from "./";` Also, the root level `jest.setup.js` file makes shallow, mount, and others globals so we don't have to import them in our test files.